pkg/cloud/services/secretsmanager/secret.go

```diff
@@ -43,9 +44,9 @@ const (
 
 var retryableErrors = []string{
 	// Returned when the secret is scheduled for deletion
-	secretsmanager.ErrCodeInvalidRequestException,
+	"InvalidRequestException",
```

Comment:
Is there really no replacement constant in SDK v2?

Reply:
sorry, I couldn’t find it.